index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/c92c970d81.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <div class="top-bar">
+            <ul>
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                </li>
+                <li>
+                    <a href="#">English</a>
+                </li>
+                <li>
+                    <a href="#">My Page</a>
+                </li>
+                <li>
+                    <a href="#">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <div class="main-bar">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="nav-top">
+                <div class="art-logo">
+                    <a href="index.html"><img src="./images/image-logo.png" alt="Art logo"></a>
+                </div>    
+                <div class="nav-nav">
+                    <ul class="link-link">
+                        <li class="nav-link">
+                            <a class="nav-home" href="#">Home</a>
+                        </li>
+                        <li class="nav-link">
+                            <a href="about.html">About</a>
+                        </li>
+                        <li class="nav-link">
+                            <a href="#">Program</a>
+                        </li>
+                        <li class="nav-link">
+                            <a href="#">Join</a>
+                        </li>
+                        <li class="nav-link">
+                            <a href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-link">
+                            <a href="#">News</a>
+                        </li>
+                        <li class="nav-link campaign-align">
+                            <a class="campaign" href="#">A&C Campaign</a>
+                        </li>
+                    </ul>
+                </div>            
```

Comment:
- Please use the `<nav>` tag in the navbar section like you did in the `<footer>` section. This is because you need to get used to HTML semantic tags.
- Open the Link to learn more about [Semantic Html](https://www.w3schools.com/html/html5_semantic_elements.asp#:~:text=What%20are%20Semantic%20Elements%3F,%3E%20%2D%20Clearly%20defines%20its%20content.)

Reply:
Thank you, @OLIPLICHE. I have fixed the issue raised.